runtime/src/main/java/org/corfudb/runtime/view/TableRegistry.java

```diff
@@ -270,9 +270,6 @@ void registerTable(@Nonnull String namespace,
                         log.warn("Protobuf file list changed for {} table {}. Prev list {}, new list {}",
                             namespace, tableName, oldRecord.getPayload().getFileDescriptorsMap().keySet(),
                             tableDescriptors.getFileDescriptorsMap().keySet());
-                        for (StackTraceElement st : Thread.currentThread().getStackTrace()) {
-                            log.debug("{}", st);
-                        }
```

Comment:
StackTrace information is also logged in `recordNewSchema`. Did you want to remove it there as well?